benchmark/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.amazon</groupId>
+    <artifactId>randomcutforest-parent</artifactId>
+    <version>1.0</version>
```

Comment:
wonder if version should be bumped?

Reply:
I've been thinking that it's not _really_ 1.0 until we push to Maven. We're also changing from an artifact ID of `random-cut-forest:1.0` for the whole package, versus `randomcutforest-core:1.0` for the core library, etc.

If you think it would be better, I could change the version numbers in this CR to something like 1.0-alpha, and we can remove the alpha just before pushing to Maven.

Reply:
yes, good point, I am good with 1.0 then. My minor concern was existing code already built on pre-change api. Since the artifacts are new, an explicit upgrade is needed.